sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipRoutingClient.cs

```diff
@@ -278,16 +390,18 @@ public virtual Response<IReadOnlyList<SipTrunk>> GetTrunks(
         /// <summary>
         /// Get List of configured <see cref="SipTrunk"/>.
         /// </summary>
+        /// <param name="expand">Expand enum.</param>
         /// <param name="cancellationToken">Optional cancellation token.</param>
         /// <returns>List of configured trunks.</returns>
         public virtual async Task<Response<IReadOnlyList<SipTrunk>>> GetTrunksAsync(
+            ExpandEnum expand,
```

Comment:
Feedback from review:
don't use ExpandEnum in public methods, let's use:
 boolean includeHealth parameter instead.